libsolidity/ast/Types.cpp

```diff
@@ -1728,10 +1745,19 @@ u256 FunctionType::externalIdentifier() const
 
 TypePointers FunctionType::parseElementaryTypeVector(strings const& _types)
 {
+	cout << "parseElementaryTypeVector " << _types << endl;
 	TypePointers pointers;
 	pointers.reserve(_types.size());
 	for (string const& type: _types)
-		pointers.push_back(Type::fromElementaryTypeName(type));
+	{
```

Comment:
No, it is needed, but it is fine for now if it does not work for all types (should throw an exception). But this might be easy to do with the other general comment I added about an elementary type name class.